.gitignore

```diff
@@ -69,3 +69,6 @@ typings/
 .vscode
 
 .config
+
+
+.cache
```

Comment:
Go for it. Maybe have it in a separate commit to readability

Reply:
done